### DISCUSSION TITLE
What is the default password of argocd?

### DISCUSSION THREAD

**Discussion by ukreddy-erwin** [argoproj/argo-cd#6663] [Q&A]
I have installed argocd on aks using below command:

`kubectl create namespace argocd`
`kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/master/manifests/install.yaml`

Then I change it to load balancer service.
`kubectl edit svc argocd-server -n argocd
`
Now, when I connect to argocd web ui, I wasm't able to connect with below credentials.

```
user: admin
password: argocd-server-9b77b6575-ts54n 
```

Password got from below command as mentioned in docs.

```
kubectl get po -n argocd
NAME                                 READY   STATUS    RESTARTS   AGE
argocd-application-controller-0      1/1     Running   0          21m
argocd-dex-server-5559bc9679-5mj4v   1/1     Running   1          21m
argocd-redis-74d8c6db65-sxbnt        1/1     Running   0          21m
argocd-repo-server-6866f58df-m59sr   1/1     Running   0          21m
argocd-server-9b77b6575-ts54n        1/1     Running   0          21m
```

Please suggest me how to login, what is the default credentials.

Even I tried resetting it using this command.
```
kubectl -n argocd patch secret argocd-secret -p '{"stringData": {
    "admin.password": "$2a$10$Ix3Pd7mywOwVWOK8eSSY0uo60V6Vf6DtZljGuLwGRHQNnWNBbOLhW",
    "admin.passwordMtime": "'$(date +%FT%T%Z)'"
  }}'
```

But getting this error:
```
Error from server (BadRequest): invalid character 's' looking for beginning of object key string
Error from server (NotFound): secrets "2021-07-08T12:59:15IST" not found
Error from server (NotFound): secrets "\n  }}" not found
```

**Comment by jannfis** [comment]
If you're using Argo CD v2.0 and higher, the initial admin password can be found in the field `.data.password` in the `argocd-initial-admin-secret` secret, as described in the docs at https://argo-cd.readthedocs.io/en/stable/getting_started/#4-login-using-the-cli

(We should really remove the old pre-2.0 instructions from there, they are confusing)

**Reply by kannamr** [reply]
@jannfis When upgraded to v2, I am not able to find the ```argocd-initial-admin-secret``` in the secrets.
